contracts/metadata/metadata.cairo

```diff
@@ -0,0 +1,647 @@
+// -----------------------------------
+//   module.Uri Library
+//   Builds a JSON array which to represent Realm metadata
+//
+// MIT License
+// -----------------------------------
+
+%lang starknet
+
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+from starkware.cairo.common.registers import get_label_location
+from starkware.cairo.common.uint256 import Uint256, uint256_unsigned_div_rem
+from starkware.cairo.common.math import unsigned_div_rem
+from starkware.cairo.common.math_cmp import is_not_zero
+from starkware.cairo.common.bool import TRUE, FALSE
+
+from contracts.settling_game.utils.game_structs import RealmData
+
+namespace Utils {
+    namespace Symbols {
+        const LeftBracket = 123;
+        const RightBracket = 125;
+        const InvertedCommas = 34;
+        const Comma = 44;
+    }
+
+    namespace ResourceNames {
+        const Wood = 1466920804;
+        const Stone = 358435745381;
+        const Coal = 1131372908;
+        const Copper = 74145906845042;
+        const Obsidian = 5720261373207339374;
+        const Silver = 91712256370034;
+        const Ironwood = 5292415032354631524;
+        const ColdIron = 4859221700940820334;
+        const Gold = 1198484580;
+        const Hartwood = 5215575688017309540;
+        const Diamonds = 4929578389782553715;
+        const Sapphire = 6008207005979341413;
+        const Ruby = 1383424633;
+        const DeepCrystal = 82685785959151284352475500;
+        const Ignium = 80708582864237;
+        const EtherealSilica = 1408709038586369043654121097880417;
+        const TrueIce = 23769746579743589;
+        const TwilightQuartz = 1713183185033713553468714470503546;
+        const AlchemicalSilver = 86962604016411212709514121790885356914;
+        const Adamantine = 308805516168419061886565;
+        const Mithral = 21789521896169836;
+        const Dragonhide = 323230868360603032446053;
+    }
+
+    namespace OrderNames {
+        const Power = 345467479410;
+        const Giants = 78517931766899;
+        const Titans = 92811900841587;
+        const Skill = 358284356716;
+        const Perfection = 379660683168662059315054;
+        const Brilliance = 313786713259670802162533;
+        const Enlightenment = 5500917626309127724772157124212;
+        const Protection = 379900278609487843651438;
+        const Twins = 362780651123;
+        const Reflection = 389104553132122754871150;
+        const Detection = 1261689176585207902062;
+        const Fox = 4616056;
+        const Vitriol = 24322796853751660;
+        const Fury = 1182102137;
+        const Rage = 1382115173;
+        const Anger = 281025144178;
+    }
+
+    namespace WonderNames {
+        const wonder_1 = 5869685279522133479430859692172194274569849; // Cathedral Of Agony
```

Comment:
Just a note - we don't have to have these magic numbers here. `const wonder_1 = 'Cathedral Of Agony'` is more readable and works the same in Cairo.